examples/specs/rect_mosaic_labelled_with_offset.vl.json

```diff
@@ -236,7 +236,7 @@
   },
   "config": {
     "view": {
-      "stroke": null
+      "stroke": ""
```

Comment:
Why empty string here?  I think

Reply:
Same for all other examples

Reply:
Tests were failing if `stroke` was not a string.